src/AgentContainer.H

```diff
@@ -74,20 +74,25 @@ public:
         : amrex::ParticleContainer< 0,
                                     0,
                                     RealIdx::nattribs,
-                                    IntIdx::nattribs> (a_geom, a_dmap, a_ba)
+                                    IntIdx::nattribs> (a_geom, a_dmap, a_ba),
+          student_counts(a_ba, a_dmap, SchoolType::total_school_type, 0)
     {
         BL_PROFILE("AgentContainer::AgentContainer");
         AMREX_ASSERT(m_num_diseases < ExaEpi::max_num_diseases);
         m_num_diseases = a_num_diseases;
         m_disease_names = a_disease_names;
 
+        student_counts.setVal(0);  // Initialize the MultiFab to zero
+
         add_attributes();
 
         {
             amrex::ParmParse pp("agent");
             pp.query("symptomatic_withdraw", m_symptomatic_withdraw);
             pp.query("shelter_compliance", m_shelter_compliance);
             pp.query("symptomatic_withdraw_compliance", m_symptomatic_withdraw_compliance);
+            pp.queryarr("student_teacher_ratios", student_teacher_ratios);
```

Comment:
@terasaki1 Can you add documentation for this new input parameter [here](https://github.com/AMReX-Codes/ExaEpi/blob/41e4f3b953ac5a40d38803ecde4a09239085e832/docs/source/usage/how_to_run.rst?plain=1#L93)? Just follow the example for the other input parameters; this will then appear [here](https://exaepi.readthedocs.io/en/latest/usage/how_to_run.html#inputs-parameters).